Sources/Dependencies/DependencyValues.swift

```diff
@@ -95,7 +95,9 @@ public struct DependencyValues: Sendable {
   /// provide access only to default values. Instead, you rely on the dependency values' instance
   /// that the library manages for you when you use the ``Dependency`` property wrapper.
   public init() {
-    _ = setUpTestObservers
+    #if DEBUG
+      _ = setUpTestObservers
+    #endif
```

Comment:
I guess this one wouldn't need to be rolled back to fix the Linux release build but I started by simply reverting the whole patch for now.